server/eb/app.py

```diff
@@ -80,8 +79,13 @@ def _before_adding_routes(app, app_config):
         web_base_url = server_config.get_web_base_url()
         if web_base_url:
             web_base_url_parse = urlparse(web_base_url)
-            allowed_origin = f"{web_base_url_parse.scheme}://{web_base_url_parse.netloc}"
-            CORS(app, supports_credentials=True, origins=allowed_origin)
+            allowed_origins = [f"{web_base_url_parse.scheme}://{web_base_url_parse.netloc}"]
+            if os.getenv('DEPLOYMENT_STAGE') in ["Staging"]:
+                allowed_origins.extend([
+                    "https://canary-cellxgene.dev.single-cell.czi.technology/",
```

Comment:
The auto-deployed canary env is https://cellxgene.canary.single-cell.czi.technology. Should we use that instead? Thank you!

Reply:
Sorry where are you seeing that? `https://canary-cellxgene.dev.single-cell.czi.technology/` is set as the web base url in the config so I think thats what we should be using?

Reply:
This is the auto deploy GH flow I was referring to!

https://github.com/chanzuckerberg/single-cell-infra/runs/3547390406?check_suite_focus=true

<img width="1134" alt="Screen Shot 2021-09-17 at 2 20 57 PM" src="https://user-images.githubusercontent.com/6309723/133854898-59eb72cc-1cd2-49b4-a201-4513bb274879.png">

I also see `canary` and `canary-hook`, so not sure if we only need one or both 😆 

https://github.com/chanzuckerberg/single-cell-infra/actions?page=3

<img width="1052" alt="Screen Shot 2021-09-17 at 2 21 48 PM" src="https://user-images.githubusercontent.com/6309723/133854996-c68ea038-704b-4a3a-b3f9-76395719727a.png">

Reply:
So canary hook is sent from the explorer repo to the single-cell-infra repo when code is merged into the main-canary branch. Single-cell-infra then kicks off the deployment in the canary step

Reply:
Ahh got it, that's why both are needed! Thanks, Madison 🙆‍♂️ 

So you're saying https://canary-cellxgene.dev.single-cell.czi.technology/ is the right URL, and the smoke test is just pointing to the wrong one (https://cellxgene.canary.single-cell.czi.technology?

Reply:
yea